### ISSUE TITLE
Why do You need READ_PHONE_STATE

### ISSUE THREAD

**Issue by piernik** [mapsplugin/cordova-plugin-googlemaps#2176]
**I'm submitting a ...**  (check one with "x")
- [x] question
- [ ] any problem or bug report
- [ ] feature request

**If you choose 'problem or bug report', please select OS:**  (check one with "x")
- [x] **Android**
- [ ] **iOS**

After installing this plugin app needs `READ_PHONE_STATE` permission.
I get feedback - people doesn't like it :/
My app is getting low score only because of that permission.
Is it necessary? What  is it for?

**Comment by wf9a5m75** [comment]
Does this plugin request the `READ_PHONE_STATE` permission?
https://github.com/mapsplugin/cordova-plugin-googlemaps/blob/4bc39caafe52815148892956197500bb44238ce0/plugin.xml#L222-L234

**Comment by wf9a5m75** [comment]
ping

**Comment by battika** [comment]
Actually I was also surprised to see that the latest APK I submitted to Play Store also required READ_PHONE_STATE. I could not figure it why, but it was definitely there.

**Comment by piernik** [comment]
I've made clean install of ionic and added google maps plugin. Then in `platforms/android/build/intermediates/manifest/full/release/AndroidManifest.xml` was added:
```
    <android:uses-permission android:name="android.permission.READ_PHONE_STATE" />
    <android:uses-permission android:name="android.permission.READ_EXTERNAL_STORAGE" />
```

**Comment by wf9a5m75** [comment]
This plugin does not request the `READ_EXTERNAL_STORAGE` and the `READ_PHONE_STATE` permissions.
Did you see the plugin.xml?

**Comment by wf9a5m75** [comment]
Check out the plugin.xml first.

https://github.com/mapsplugin/cordova-plugin-googlemaps/blob/4bc39caafe52815148892956197500bb44238ce0/plugin.xml

**Comment by makers-mark** [comment]
@piernik  & @battika 

It looks to be a play services issue according to [this stack overflow thread](https://stackoverflow.com/questions/39668549/why-has-the-read-phone-state-permission-been-added). You should read all of this, even the replies, and a potential workaround for now:
```
<manifest ...
    xmlns:tools="http://schemas.android.com/tools"
    ... >

<uses-permission
    android:name="android.permission.READ_PHONE_STATE"
    tools:node="remove" />
```

**Comment by battika** [comment]
Thanks, so the new Play Services version is causing READ_PHONE_STATE to be a required feature.

**Comment by makers-mark** [comment]
@wf9a5m75 
@battika 
@piernik 
[It is fixed.](https://developers.google.com/android/guides/releases)

**Comment by piernik** [comment]
@wf9a5m75 
@makers-mark 
I've installed newest version of this plugin `2.3.0-rc1`, deleted `platform` folder` and when I build app `android.permission.READ_PHONE_STATE` is still there.
Can You confirm that `READ_PHONE_STATE` is not required?

**Comment by wf9a5m75** [comment]
Did I write `cordova plugin add cordova-plugin-googlemaps#2.3.0-rc1` in somewhere?

**Comment by piernik** [comment]
In readme file: https://github.com/mapsplugin/cordova-plugin-googlemaps/commit/8b5bb66a4c05e403f9e4360fcd4e1c937d518b51 line 13

**Comment by piernik** [comment]
@wf9a5m75 You released 2.2.9 This version is dependent on google play services 15?

**Comment by wf9a5m75** [comment]
I released v2.3.0

**Comment by piernik** [comment]
Oh - great!
Thanks
